src/scripts/main.js

```diff
@@ -1,3 +1,85 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const message = document.createElement('div');
+
+// message.style.top = `30px`;
+// message.style.right = `30px`;
+
+const urlList
+  = 'https://mate-academy.github.io/phone-catalogue-static/api/phones.json';
```

Comment:
```suggestion
  = 'https://mate-academy.github.io/phone-catalogue-static/api/';
```
Можна використовувати універсальний url для цієї константи і використати її в методі getPhonesDetails:
```
fetch(`${urlList}/${id}.json`)